docs/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Overview
+toc: true
+weight: 400
+indent: true
+---
+
+# Overview
+
+Crossplane introduces multiple building blocks that enable you to provision,
+publish, and consume infrastructure using the Kubernetes API. These individual
+concepts work together to allow for powerful separation of concern between
+different personas in an organization, meaning that each member of a team
+interacts with Crossplane at an appropriate level of abstraction.
+
+![Crossplane Concepts]
```

Comment:
Might move this **after** the key concepts (i.e. after the OAM section) are introduced and then provide a **text walkthrough of the diagram** building on the concepts introduced on the docs/readme.md
* **Provision & manage cloud infrastructure with kubectl**
  * [Install Crossplane] to provision and manage cloud infrastructure and
    services from any Kubernetes cluster.
  * Provision infrastructure primitives from any provider ([GCP], [AWS],
    [Azure], [Alibaba], on-prem) and use them alongside existing application
    configurations.
  * Version, manage, and deploy with your favorite tools and workflows that
    you’re using with your clusters today.

* **Publish custom infrastructure resources for your applications to use**
  * Define, compose, and publish your own [infrastructure resources] with
    declarative YAML, resulting in your own infrastructure CRDs being added to
    the Kubernetes API for applications to use.
  * Hide infrastructure complexity and include policy guardrails, so
    applications can easily and safely consume the infrastructure they need,
    using any tool that works with the Kubernetes API.
  * Consume infrastructure resources alongside any Kubernetes application to
    provision and manage the cloud services they need with Crossplane as an
    add-on to any Kubernetes cluster.

* **Deploy applications using a team-centric approach with OAM**
  * Define cloud native applications and the infrastructure they require with
    the Open Application Model ([OAM]).
  * Collaborate with a team-centric approach with a strong separation of
    concerns:
    * Infrastructure operators - provide infrastructure and services for
      applications to consume
    * Application developers - build application components independent of
      infrastructure
    * Application operators - compose, deploy, and run application
      configurations
  * Deploy application configurations from app delivery pipelines or GitOps
    workflows, using the proven Kubernetes declarative model.

Reply:
So basically a text walkthrough of the diagram showing more concretely, using the example presented the diagram, for how Crossplane enables you to do all these things.

Reply:
@prasek that run-down is pretty much verbatim what is on the docs `README.md`, which is the first page that folks are going to see. Are we wanting to reproduce it here as well?

Reply:
No not repro, just a walkthrough of the diagram touching on the points covered in the readme, i.e. who does what. Essentially the voiceover for the summary diagram if you were going to explain it to someone.